CODEOWNERS changes & renovate reviews

armenzg opened the pull request.

armenzg commented:
@sarah-clements @camd r?

sarah-clements commented on .renovaterc:
```diff
@@ -14,7 +14,7 @@
   ],
   "labels": ["dependencies", "javascript"],
-  "reviewers": ["armenzg"],
+  "reviewers": ["sclements"],
```
Ah, you beat me to it. I had a pr for this but hadn't merged it yet :) This should be my github handle right? If so, it's 'sarah-clements'.

sarah-clements commented on CODEOWNERS:
```diff
@@ -1,14 +1,8 @@
 
-/.travis.yml                @armenzg
-/app.json                   @armenzg
-/docker-compose.yml         @armenzg
```
Maybe we should keep some of these but change it to treeherder admins.

armenzg commented:
Hi Sarah,
I've addressed your concerns. Could you please merge if it's satisfying?

Would you mind to give me some access to the repo? (e.g. assigning myself or even write pending admin merging)
I have not yet decided if I will contribute long term but I think it would not be too much concern if I had *some* access.

sarah-clements commented:
> Hi Sarah,
> I've addressed your concerns. Could you please merge if it's satisfying?
> 
> Would you mind to give me some access to the repo? (e.g. assigning myself or even write pending admin merging)
> I have not yet decided if I will contribute long term but I think it would not be too much concern if I had _some_ access.

I've sent you an invite with write access. Any contributions you'd like to make would be most welcome :)

armenzg commented:
Thank you Sarah!
